CHANGES.txt

```diff
@@ -9,6 +10,8 @@ NOTE: The 4.0.0 Release of SCons will drops Python 2.7 Support
 RELEASE 4.1.0.devyyyymmdd - Mon, 04 Jul 2020 16:06:40 -0700
 
   From Rob Boehne:
+    - Fix fortran tools to set SHFORTRAN variables to $FORTRAN (also for similar variables
```

Comment:
List all the tools you updated please. Add something like "If you were depending on setting FORTRAN not changing SHFORTRAN (or equivalent for fortran version you are using), this change will break that."